common/src/main/java/tachyon/Constants.java

```diff
@@ -144,6 +144,12 @@
       "tachyon.integration.master.resource.mem";
   public static final String INTEGRATION_YARN_ONE_WORKER_PER_HOST =
       "tachyon.integration.yarn.one.worker.per.host";
+  public static final String INTEGRATION_YARN_WORKER_CONTAINER_REQUEST_ROUNDS_MAX =
+      "tachyon.integration.yarn.worker.container.request.rounds.max";
+  public static final String INTEGRATION_YARN_CONTAINER_REQUEST_RETRY_COUNT =
+      "tachyon.integration.yarn.container.request.retry";
+  public static final String INTEGRATION_YARN_CONTAINER_REQUEST_TIMEOUT_S =
```

Comment:
maybe rename this property to be `tachyon.integration.yarn.container.request.timeout.seconds` (and accordingly the const string name`